indigo_social/models.py

```diff
@@ -1,9 +1,32 @@
 # -*- coding: utf-8 -*-
+import os
+
 from django.contrib.auth.models import User
+from django.core.files import File
 from django.db import models
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 
+from urllib.request import urlretrieve, urlcleanup
+
+from allauth.account.signals import user_signed_up
+from allauth.socialaccount.models import SocialLogin, SocialAccount
+from allauth.socialaccount.signals import pre_social_login, social_account_added, social_account_updated
+
+
+def user_profile_photo_path(instance, filename):
+    return 'avatars/{}/{}'.format(instance.id, os.path.basename(filename))
```

Comment:
I don't even think it needs a filename. We don't want to keep old photos around -- just save to `avatars/99` for UserProfile with pk=99.